content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -867,6 +867,47 @@ is disabled, `.spec.completions` is immutable.
 Use cases for elastic Indexed Jobs include batch workloads which require 
 scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
 
+### Recreation only when Pod has been fully terminated
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+For distributed jobs, it is common to have many replicas running in a job.  
+Some frameworks will require uniqueness for the pod index.  
+In cases like this, it is beneficial to only recreate pods
+based once the pod has been terminated.
+
+When the `JobPodReplacementPolicy`[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is set, you will be able to specify `PodReplacementPolicy` to `Failed`.  
+This will only recreate pods once they are terminated.  
+Default behavior will be to recreate upon deletion (`DeletionTimestamp != nil`).
+
+```yaml
+kind: Job
+metadata:
+  name: new
+  ...
+spec:
+  PodReplacementPolicy: Failed
+  ...
+```
+
+To improve debuggability/observability for pods that are being deleted (`DeletionTimestamp != nil`) we include a new field in the JobStatus called `terminating`.  
+This will report the number pods that are currently terminating and is easily viewable in the status.
+
+```shell
+kubectl get jobs/myjob -o yaml
+```
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+# .metadata and .spec omitted
+status:
+  terminating: 1 # if pod is terminating
+```
+
+In cases where `PodFailurePolicy` is enabled, we will assume `Failed` as the default and only allowed value.  
```

Comment:
I think it would also be beneficial to x-reference this section (with links) from the pod failure policy, and vice versa.